ethcore/evm/src/tests.rs

```diff
@@ -19,17 +19,22 @@ use std::str::FromStr;
 use std::hash::Hash;
 use std::sync::Arc;
 use std::collections::{HashMap, HashSet};
-use rustc_hex::FromHex;
 use ethereum_types::{U256, H256, Address};
 use vm::{self, ActionParams, ActionValue, Ext};
 use vm::tests::{FakeExt, FakeCall, FakeCallType, test_finalize};
 use factory::Factory;
 use vmtype::VMType;
+use hex_literal::hex;
 
 evm_test!{test_add: test_add_int}
 fn test_add(factory: super::Factory) {
 	let address = Address::from_str("0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6").unwrap();
-	let code = "7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff01600055".from_hex().unwrap();
+	// 7f       PUSH32 ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff
+	// 7f       PUSH32 ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff
+	// 01       ADD
+	// 60 00    PUSH 0
+	// 55       SSTORE
```

Comment:
Would be nice to have a macro/builder for bytecode instead of having comments, but that's outside of the scope of the PR.

Reply:
In the meantime https://ethervm.io/decompile can be helpful.